src/lib/menu/index.ts

```diff
@@ -1 +1,8 @@
 export * from './menu';
```

Comment:
I get the following error message while compiling my project (with Angular-CLI Webpack):
```
WARNING in ./~/@angular/material/module.js
44:4 export 'MdMenuModule' was not found in './menu/index'
```

Running the latest version: **2.0.0-alpha.11-1**. It wasn't there before.